hail/python/test/hail/table/test_table.py

```diff
@@ -1838,36 +1848,6 @@ def test_can_process_wide_tables(width):
     ht.annotate_globals(g=ht.collect(_localize=False))._force_count()
 
 
-def create_width_scale_files():
```

Comment:
Someone else on query might want to weigh in on why this is here. It does appear to be unused.

Reply:
will check on zulip!

Reply:
important for generating some test files, leaving it in with dan's change that makes it actually work again!